lib/MetaCPAN/Web/Controller/Author.pm

```diff
@@ -129,6 +132,29 @@ sub releases : Chained('root') PathPart Args(0) {
     );
 }
 
+sub _calc_aggregated {
+    my ( $self, $releases ) = @_;
+
+    my @aggregated;
+    my $latest = $releases->[0];
+    my $last;
+
+    for my $rel ( @{$releases} ) {
+        my ( $canon_rel, $canon_lat ) = map {
+            DateTime::Format::ISO8601->parse_datetime($_)
+                ->strftime("%Y%m%d%H%M%S")
+        } ( $rel->{date}, $latest->{date} );
+        $latest = $rel if $canon_rel > $canon_lat;
+
+        next if $last and $last eq $rel->{distribution};
+        $last = $rel->{distribution};
+        next unless $rel->{name};
+        push @aggregated, $rel;
+    }
+
+    return +[ \@aggregated, $latest ];
```

Comment:
I can see it for 
```perl
sub { +{ foo => 'bar' } }
```

since that makes the braces easier to read, but with the array reference it mostly confuses me.  :)